scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/service/ScalingScheduleManager.java

```diff
@@ -0,0 +1,162 @@
+package org.cloudfoundry.autoscaler.scheduler.service;
+
+import java.sql.Time;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.cloudfoundry.autoscaler.scheduler.dao.ScheduleDao;
+import org.cloudfoundry.autoscaler.scheduler.entity.ScheduleEntity;
+import org.cloudfoundry.autoscaler.scheduler.rest.model.ApplicationScalingSchedules;
+import org.cloudfoundry.autoscaler.scheduler.rest.model.SpecificDateSchedule;
+import org.cloudfoundry.autoscaler.scheduler.util.DateHelper;
+import org.cloudfoundry.autoscaler.scheduler.util.JobScheduleTypeEnum;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * Service class to persist the schedule entity in the database and create
+ * scheduled job.
+ * 
+ * @author Fujitsu
+ *
+ */
+@Service
+public class ScalingScheduleManager {
```

Comment:
I would prefer the Service deal with the entities rather than do the conversion from the REST model to the entity model. We get better control and understanding with the Entity.